app/main.py

```diff
@@ -2,0 +1,47 @@
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None :
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None :
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        for skill in ["JavaScript", "HTML", "CSS"]:
+            self.learn_skill(skill)
```

Comment:
Consider using `self.skills.extend(['JavaScript', 'HTML', 'CSS'])` instead of the loop to initialize skills for better performance and readability.